config/default/eu_cookie_compliance.settings.yml

```diff
@@ -0,0 +1,41 @@
+cookie_lifetime: 100
+domain: ''
+domains_option: 1
+domains_list: ''
+eu_only: false
+exclude_paths: "/admin\r\n/admin/*\r\n/batch\r\n/node/add*\r\n/node/*/*\r\n/user/*/*"
+exclude_admin_theme: true
+export:
+  folder: custom
+langcode: cs
+popup_agreed:
+  value: "<h2>Thank you for accepting cookies</h2>\r\n\r\n<p>You can now hide this message or find out more about cookies.</p>\r\n"
```

Comment:
preklad?

Reply:
Nemáme tuhle funkci aktivní, ale přeložil jsem to.